azplugins/BondEvaluatorDoubleWell.h

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2018-2020, Michael P. Howard
+// This file is part of the azplugins project, released under the Modified BSD License.
+
+// Maintainer: astatt
+
+/*! \file BondEvaluatorDoubleWell.h
+    \brief Defines the bond evaluator class for a double well potential
+*/
+
+#ifndef AZPLUGINS_BOND_EVALUATOR_DOUBLE_WELL_H_
+#define AZPLUGINS_BOND_EVALUATOR_DOUBLE_WELL_H_
+
+#ifndef NVCC
+#include <string>
+#endif
+
+#include "hoomd/HOOMDMath.h"
+
+#ifdef NVCC
+#define DEVICE __device__
+#else
+#define DEVICE
+#endif
+
+namespace azplugins
+{
+namespace detail
+{
+
+//! Class for evaluating the double well bond potential
+/*!
+This bond potential follows the functional form
+\f{eqnarray*}
+V_{\mathrm{DW}}(r)  =   \frac{V_{min}}{b^4} \left[ \left( r - a/2 \right)^2 -b^2 \right]^2
+\f}
+and has two minima at r = 1/2(a +/- 2b), seperated by a maximum at 1/2a of height V_max.
```

Comment:
See my comments in the py file for writing all quantities in these docs like `a/2` rather than `1/2a`, if this is what you intended.